go.mod

```diff
@@ -1,6 +1,7 @@
 module github.com/cloudflare/ebpf_exporter/v2
 
 go 1.22
+toolchain go1.22.10
```

Comment:
https://github.com/grpc-ecosystem/grpc-gateway/pull/4919/files#r1890941129 due to included `grpc-gateway` bump.